scripts/api/data/licenses/licenseAGPL-3.0-or-later.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
@philippconzett can you please add a release note for this PR? (I'm just adding a comment here because it's the first line under "files changed".